Pipelines/Templates/build-Solution.yml

```diff
@@ -36,10 +36,21 @@ steps:
   condition: and(succeeded(), ne('${{parameters.buildType}}', 'Unmanaged'))
 
 - template: Hooks\build-solution-manipulate-source-pre-hook.yml
-  
+
+- pwsh: |
+      . "$env:POWERSHELLPATH/code-first-functions.ps1"
+      install-pac-and-authenticate "${{parameters.serviceConnectionUrl}}" "$(ClientId)" "$(ClientSecret)" "$(TenantID)"

```

Comment:
We shouldn't need to install pac anymore.  It is on disk now and made available viable the installer task.

Here is an example script of how to get pac on the agent

$agentOS = "$(Agent.OS)"

if ($agentOS -eq "Linux") {
  $pacPath = $env:POWERPLATFORMTOOLS_PACCLIPATH + "/pac_linux/tools"
  $env:PATH = $env:PATH + ":" + $pacPath #note colon delimeter in path for linux
}
else {
  $pacPath = $env:POWERPLATFORMTOOLS_PACCLIPATH + "\pac\tools"
  $env:PATH = $env:PATH + ";" + $pacPath #note semi-colon delimeter in path for windows
}

pac

Reply:
@devkeydet Removed the Install pac logic and replaced with _$pacPath = $env:POWERPLATFORMTOOLS_PACCLIPATH + "\pac\tools"_